pkgs/applications/emulators/86box/update.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p jq nix-prefetch-github
+
+set -euo pipefail
+
+cd $(readlink -e $(dirname "${BASH_SOURCE[0]}"))
+
+latest_release=$(curl --silent https://api.github.com/repos/86Box/86Box/releases/latest)
+version=$(jq -r '.tag_name' <<<"$latest_release" | cut -c2-)
+main_hash=$(nix-prefetch-github --json --rev "v$version" 86Box 86Box | jq -r '.hash')
+roms_hash=$(nix-prefetch-github --json --rev "v$version" 86Box roms | jq -r '.hash')
+
+cat >sources.nix <<EOF
+# Generated by ./update.sh - do not edit!
+{
+  version = "$version";
+  hash = "$main_hash";
+  romsHash = "$roms_hash";
+}
+EOF
```

Comment:
You can still get rid of the intermediate file using `update-source-version`:

```suggestion
#!nix-shell -i bash -p jq nix-prefetch-github common-updater-scripts

set -euo pipefail

latest_release=$(curl --silent https://api.github.com/repos/86Box/86Box/releases/latest)
version=$(jq -r '.tag_name' <<<"$latest_release" | cut -c2-)
main_hash=$(nix-prefetch-github --json --rev "v$version" 86Box 86Box | jq -r '.hash')
roms_hash=$(nix-prefetch-github --json --rev "v$version" 86Box roms | jq -r '.hash')

update-source-version _86Box "_$version" "$main_hash"
update-source-version _86Box "$version" "$roms_hash" --source-key=roms
```